app/utils/broadcast.py

```diff
@@ -301,3 +301,20 @@ def get_centroid_if_postcode_in_db(postcode, form):
         form.postcode.process_errors.append("Enter a postcode within the UK")
     else:
         return get_centroid(area)
+
+
+def format_area_name(area_name):
+    if area_name.endswith(", City of"):
+        return f"City of {area_name[:-9]}"
+    elif area_name.endswith(", County of"):
+        return f"County of {area_name[:-11]}"
+    else:
+        return area_name
+
+
+def format_areas_list(areas_list):
+    return (
+        [format_area_name(area.name) for area in areas_list]
+        if (len(areas_list) == 1 or type(areas_list) is not CustomBroadcastAreas)
+        else [format_area_name(area) for area in areas_list.items]
+    )
```

Comment:
This is very awkward to read.  There's no real reason to try and cram everything onto one line and it just makes things harder to understand, I had to read this 3-4 times just to work out what was going on.

Also I suspect that if you have CustomBroadcastAreas with a length of one this will break because it'll revert to CustomBroadcastArea functionality but will still be trying to access an array.

I'd be tempted to
```
if type(areas_list) is CustomBroadcastAreas
    return [format_area_name(area.name) for area in areas_list]
return [format_area_name(area) for area in areas_list.items]
```
if you wanted to do belt and bracers you could

```
if type(areas_list) is CustomBroadcastAreas
    return [format_area_name(area.name) for area in areas_list]
else if type(areas_list) is CustomBroadcastAreas
    return [format_area_name(area) for area in areas_list.items]
# Either throw an error, an empty string or areas_list unchanged here
```

You also want unit tests for both of these new functions